src/plugins/vis_type_table_new/public/components/table_vis_component.tsx

```diff
@@ -51,8 +52,12 @@ export const TableVisComponent = ({
       const colIndex = columns.findIndex((col) => col.id === columnId);
       const column = columns[colIndex];
       // use formatter to format raw content
-      // this can format date and percentage data
-      const formattedContent = column.formatter.convert(rawContent, 'text');
+      // this can format url, date and percentage data
```

Comment:
nit - this list is not exhaustive; it can actually format lots of other data types too

Reply:
will modify the comment msg